docker-compose.yml

```diff
@@ -42,6 +42,7 @@ services:
     command: >
       bash -c "
       python3 manage.py migrate &&
+      python3 manage.py seed &&
```

Comment:
This means any existing db would be seeded and we need to override the command for production. While migrate is supposed to be run anytime without harm, imho test data should not be added everytime we migrate, no?

Reply:
I agree with @maltaesousa ,
maybe add an `docker compose exec` command in the readme to be executed the first time or for testing, but don't put it directly at every startup.

Reply:
Added a note and removed seeding during startup